External/FEXCore/Source/Interface/Context/Context.h

```diff
@@ -71,6 +72,14 @@ namespace FEXCore::Context {
 
   class ContextImpl final : public FEXCore::Context::Context {
     public:
+      void *operator new(size_t size) {
```

Comment:
Why do we need to add these here?

Reply:
Removed until I can validate again that it is necessary.